### PR TITLE
Added unwrap function

### PR DESCRIPTION
**Pull request by 5saviahv**
Should solve #850

### CODE REVIEW

File: lib/api/manipulation.js [hunk 1]
```diff
@@ -363,6 +363,47 @@ exports.wrapInner = _wrap(function (el, elInsertLocation, wrapperDom) {
   updateDOM(wrapperDom, el);
 });
 
+/**
+ * The .unwrap() function, removes the parents of the set of matched elements
+ * from the DOM, leaving the matched elements in their place.
+ *
+ * @example <caption>without selector</caption>
+ *   const $ = cheerio.load(
+ *     '<div id=test>\n  <p>Hello</p></div>\n  <div><p>World</p>\n</div>'
```
Comment (fb55) [inline lib/api/manipulation.js:372]: This example doesn't seem entirely correct. Why wouldn't `#test` be deleted?

File: lib/api/manipulation.js [hunk 2]
```diff
@@ -363,6 +363,47 @@ exports.wrapInner = _wrap(function (el, elInsertLocation, wrapperDom) {
...
+ *     '<div id=test>\n  <p>Hello</p>\n  <b><p>World</p></b>\n</div>'
+ *   );
+ *   $('#test p').unwrap('b');
+ *
+ *   //=> <div id=test>
+ *   //     <p>Hello</p>
+ *   //     <p>World</p>
+ *   //   </div>
+ *
+ * @param {string} selector - A selector to check the parent element against. If
```
Comment (fb55) [inline lib/api/manipulation.js:392]: Mark this as optional?

```suggestion
 * @param {string} [selector] - A selector to check the parent element against. If
```

File: lib/api/manipulation.js [hunk 3]
```diff
@@ -363,6 +363,47 @@ exports.wrapInner = _wrap(function (el, elInsertLocation, wrapperDom) {
...
+ *     an element's parent does not match the selector, the element won't be unwrapped.
+ * @see {@link https://api.jquery.com/unwrap/}
+ * @returns {Cheerio} The instance itself, for chaining.
+ */
+exports.unwrap = function (selector) {
+  var self = this;
+  this.parent(selector)
```
Comment (fb55) [inline lib/api/manipulation.js:399]: I see where the motivation for the parent fix came from 😄

**Review by fb55 — COMMENTED** [review]
Thanks for looking into this! Two minor issues, otherwise this should be good to go.

File: lib/api/manipulation.js (hunk 3 above)
Comment (5saviahv) [reply lib/api/manipulation.js:400]: true, small but nasty bug

File: lib/api/manipulation.js (hunk 1 above)
Comment (5saviahv) [reply lib/api/manipulation.js]: You are correct and I updated example, sorry. Second one is correct.

**Comment by fb55** [comment]
Awesome, thanks a lot!

**Pull request merged**